JunStack.java

```diff
@@ -200,67 +33,30 @@ public void push(String value) {
     public String pop(){
         if (trackStackPosition > 0) {
            String toPop = this.stack[trackStackPosition - 1];
-           System.out.println("Popping String " + toPop + " at index: " + (trackStackPosition - 1));
-           
+
            this.stack[trackStackPosition - 1] = null;
            trackStackPosition -= 1;
            
            return toPop; 
         } else {
-            System.out.println("Stack is empty.");
-            return "Stack is empty.";
+            return null;
         }
     }
     
     public boolean isEmpty(){
-        if (trackStackPosition == 0){
-            return true;
-        } else {
-            return false;
-        }
+       return trackStackPosition == 0;
     }
-    
-    public boolean isFull(){
-        if (trackStackPosition == stack.length){
-            return true;
-        } else {
-            return false;
-        }
-    }
-    
-    public String peek(){
+
+    public String peep(){
         if (trackStackPosition > 0){
             return this.stack[trackStackPosition - 1];
         } else {
-            return "Stack is empty.";
-        }
-    }
-    
-    public void change(int index, String replacement){
-        if (replacement.length() != 1) {
-            System.out.println("Not valid input. Must be 1 character only.");
-            return;
-        }
-        this.stack[index] = replacement;
-    }
-    
-    public void display(){
-        if (trackStackPosition > 0) {
-            for (int i = trackStackPosition; i >= 0; i--){
-                System.out.println(this.stack[i]);
-            } 
-        } else {
-            System.out.println("Can't display. Stack is empty.");
+            return null;
         }
     }
-    
-    public void totalPop(){
-        if (trackStackPosition == 0){
-            return;
-        } else {
-            while (trackStackPosition > 0) {
-                pop();
-            }
-        }
+
+    public void clear(){

```

Comment:
you don't need to call a method over and over to accomplish this, just nullify dis bit

Reply:
ok i did it in the context of seeing the thing go pop (for fun)